client-guide/heartbeat-meeting.html.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Heartbeat meeting
+meta_description: >
+  TODO
+---
+
+We aim to have this meeting quarterly. You will talk with our Team lead about our relationship.
+
+## Goals
+
+The main goals of this recurring event are:
+
+- understand if our relationship is strong;
+- understand what's not going well in the collaboration;
+- understand if you need any adjustment to the team (additional team members, particular skill sets, etc.) in the next period;
+
+## Template
+
+You can use the following questions to understand the meeting content and to prepare for the discussion:
+
+- Are you generally satisfied with our collaboration?
+- Which kind of growth do you expect in the next quarter? What about one year?
+- Do you have a roadmap with the next things to do? Can you share it with us in order to understand together if the current team can deliver everything in time or if it's under/over-staffed for what's coming next?
+- Do you think someone in the team is currently not performing well or is not a perfect fit with the task they use to deliver?
```

Comment:
I don't remember the exact wording, but in the past I suggested we use positive rather than negative language here. So probably something like "Are all team members performing up to your expectations?".